remoteit/desktop
# [DESK-535][DESK-536] Fix oob errors on remote.itPi

### Description

Fix oob errors on remote.itPi
Move the logic out of sync function
update to use raspbian specific wifi check
add try catch to protect against failure
this should also fix issue DESK-536 found on windows

### Checklist


### Test plan

1. Load on remote.itPI
2. Verify app launch sucessfully
3. verify OOB functionality

### Ticket

https://remoteit.atlassian.net/browse/DESK-535
